src/wordy/ast/BlockNode.java

```diff
@@ -60,4 +63,20 @@ protected String describeAttributes() {
         return "(%d %s)"
             .formatted(statements.size(), statements.size() == 1 ? "child" : "children");
     }
+
+    @Override
+    protected void doRun(EvaluationContext context) {
+        for (var statement: statements) {
+            statement.run(context);
+        }
+    }
+
+    @Override
+    public void compile(PrintWriter out) {
+        out.print("{");
```

Comment:
My code is the same as Paul's, except I didn't use println(). It would have made it more human-readable if I did that, but it isn't necessary for it to work.